src/runtime/components/LitWrapperServer.ts

```diff
@@ -59,21 +60,20 @@ export default defineComponent({
       }
     },
 
-    getAttributesToRender() {
-      if (this.renderer.element.attributes) {
-        return Object.fromEntries(
-          this.renderer.element.attributes.map((attribute) => [attribute.name, attribute.value])
+    getShadowContents(): string | undefined {
+      if (this.renderer) {
+        return this.iterableToString(
+          this.renderer.renderShadow({
+            elementRenderers: [],
```

Comment:
Do we need these to be explicit? Since these seem to be the defaults. 
https://github.com/lit/lit/blob/main/packages/labs/ssr/src/lib/render-lit-html.ts#L530

Also, we are overriding the `elementRenderers` to be an empty array. Not sure what the consequence of that will be.

Reply:
This was a suggestion from the TS compiler as it didn't like having an empty object passed in

Reply:
Looks like the override of `elementRenderers` does not impact anything. So 👍

Reply:
I wonder if you might need to pass in `[LitElementRenderer]` here. That could affect rendering of nested Lit components.

Reply:
Awesome!